{
  "comments": [
    {
      "key": {
        "uuid": "9abe1de1_a31d5811",
        "filename": "modules/dci_file.py",
        "patchSetId": 2
      },
      "lineNbr": 199,
      "author": {
        "id": 405
      },
      "writtenOn": "2017-10-17T12:12:16Z",
      "side": 1,
      "message": "easier to read:\n\n        if module.params[\u0027path\u0027]:\n            file_path \u003d module.params[\u0027path\u0027]\n            if not os.path.exists(file_path):\n                module.fail_json(msg\u003d\u0027%s: No such file\u0027 % file_path)\n            kwargs[\u0027file_path\u0027] \u003d file_path\n        else:\n            kwargs[\u0027content\u0027] \u003d module.params[\u0027content\u0027]",
      "range": {
        "startLine": 193,
        "startChar": 0,
        "endLine": 199,
        "endChar": 56
      },
      "revId": "12eec631e14c704fac3d7c27d6d7dc29b8725583",
      "serverId": "6c1dc8ef-8b94-40e4-bd83-c2359d45ecc0",
      "unresolved": false
    }
  ]
}